docs/simulators/CaNS.md

```diff
@@ -0,0 +1,36 @@
+# CaNS
+
+[CaNS](https://github.com/CaNS-World/CaNS)(Canonical Navier-Stokes) is a code
+for massively-parallel numerical simulations of fluid flows. It aims at solving
+any fluid flow of an incompressible, Newtonian fluid that can benefit from a
+FFT-based solver for the second-order finite-difference Poisson equation in a 3D
+Cartesian grid.
+
+CaNS has been built using OpenMPI 4.1.2 and FFTW 3.3.8. We're currently operating on CaNS version 2.3.4 for CPU, with a GPU version slated for release in the near future.
+
+## Example
+
+
+```python
+import inductiva
+
+# Set simulation input directory
+input_dir = inductiva.utils.download_from_url(
+    "https://storage.googleapis.com/inductiva-api-demo-files/"
+    "cans-input-example.zip", unzip=True)
+
+# Initialize the Simulator
+cans = inductiva.simulators.CaNS()
+
+# Run simulation with config files in the input directory
+task = cans.run(input_dir=input_dir, 
+                 sim_config_filename="input.nml",
+                 n_vcpus=4)
+
+task.wait()
+task.download_outputs()
+
+```
+
+**Closing Notes**: CaNS requires input files to include a designated 'data' folder
+for storing simulation outputs.
```

Comment:
Why leave this to the user? Why not have the backend create the `data_folder` to store the outputs. We already to this for the schism simulator. See the `schism` simulator in the backend

Reply:
I've decided to delegate this to the user since the simulator operates under this principle. My intention is to keep the backend as versatile as possible, avoiding the creation of a 'special' case for just one or two simulators.

As per the documentation, "data will be written by default to a folder named data/. This folder should be located where the executable is run, typically in the run/ folder".